src/patients/diagnoses/ViewDiagnoses.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+
+import DiagnosisTable from './DiagnosisTable'
+
+const ViewDiagnoses = () => {
+  const { id } = useParams()
+
+  return <DiagnosisTable patientId={id} />
```

Comment:
```suggestion
  const { id: patientId } = useParams()

  return <DiagnosisTable patientId={patientId} />
```

I think we should rename this variable to `patientId` since `id` is ambiguous in this context.